generators/client/templates/angular/src/main/webapp/app/core/date/datepicker-adapter.ts.ejs

```diff
@@ -20,21 +20,20 @@
  */
 import { Injectable } from '@angular/core';
 import { NgbDateAdapter, NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
-import { Moment } from 'moment';
-import * as moment from 'moment';
+import * as dayjs from 'dayjs';
 
 @Injectable()
-export class NgbDateMomentAdapter extends NgbDateAdapter<Moment> {
-    fromModel(date: Moment): NgbDateStruct {
-        if (date && moment.isMoment(date) && date.isValid()) {
+export class NgbDateDayjsAdapter extends NgbDateAdapter<dayjs.Dayjs> {
+    fromModel(date: dayjs.Dayjs): NgbDateStruct {
+        if (date && dayjs.isDayjs(date) && date.isValid()) {
             return { year: date.year(), month: date.month() + 1, day: date.date() };
         }
         // ! can be removed after https://github.com/ng-bootstrap/ng-bootstrap/issues/1544 is resolved
         return null!;
     }
 
-    toModel(date: NgbDateStruct): Moment {
+    toModel(date: NgbDateStruct): dayjs.Dayjs {
         // ! after null can be removed after https://github.com/ng-bootstrap/ng-bootstrap/issues/1544 is resolved
```

Comment:
https://github.com/ng-bootstrap/ng-bootstrap/issues/1544 has been fixed in ng-bootstrap 6.0.1, so probably "! after null can be removed"?

Reply:
I removed it, it's ok now, thanks